net/geth/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2018 OpenWrt.org
```

Comment:
no. you don't have copyright assignment for openwrt.  either a) drop this line completely, or b) put your own copyright.

Reply:
ok.